tests/integration/test_select.py

```diff
@@ -80,6 +80,7 @@ def test_select_of_select(c, df):
 
 
 def test_select_of_select_with_casing(c, df):
+    c.set_config(("dask.sql.identifier.case.sensitive", True))
```

Comment:
Was the default behavior of dask-sql before to have case sensitivity be set true? In that case, should that be the option we default to if it's left unspecified? Right now it looks like we would default to false.

Reply:
Yeah, probably a good idea